sonar-orchestrator/src/main/java/com/sonar/orchestrator/db/DefaultDatabase.java

```diff
@@ -359,7 +378,7 @@ DefaultDatabase executeDdl(Connection connection, String... ddls) {
     try {
       stmt = connection.createStatement();
       for (String ddl : ddls) {
-        LOG.debug("Execute: {}", ddl);
+        LOG.info("Execute: {}", ddl);
```

Comment:
please leave this as debug

Reply:
I thought that you have done this purposely. I will change it.